_topic_maps/_topic_map_rosa.yml

```diff
@@ -857,7 +857,7 @@ Topics:
 - Name: Available services
   File: rosa-available-services
 ---
-Name: Storage
```

Comment:
This is the issue that's causing the build to fail. If you add this line back in, it should work.